Example/Gemfile.lock

```diff
@@ -1,3 +1,6 @@
+GEM
+  specs:
+
```

Comment:
Was this autogenerated? I don't know much about Ruby but it seems weird that there are two `GEM` elements and this one is empty.

Reply:
Yeah, it generated this when I ran `bundle install` 🤷‍♂️